src/tracing/default.rs

```diff
@@ -2,17 +2,26 @@ use std::collections::HashSet;
 
 use common::ext::OptionExt;
 use serde::{Deserialize, Serialize};
-use tracing_subscriber::prelude::*;
-use tracing_subscriber::{filter, fmt, registry};
+use tracing_subscriber::fmt::{self};
+use tracing_subscriber::{filter, registry, Layer};
 
 use super::*;
 
+#[derive(Default, Clone, Debug, Eq, PartialEq, Deserialize, Serialize)]
+#[serde(rename_all = "lowercase")]
```

Comment:
Nitpick: not necessary now, but probably a better default if we ever add more variants here.

```suggestion
#[serde(rename_all = "snake_case")]
```